src/openforms/typing.py

```diff
@@ -3,7 +3,8 @@
 import datetime
 import decimal
 import uuid
-from typing import TYPE_CHECKING, Any, NewType, Protocol, Sequence
+from collections.abc import Sequence
```

Comment:
I thought it was the same if we import it from typing

Reply:
they're aliases, but the canonical place is `collections.abc`.

In the future, this will be addressed with the Ruff `UP` rules too I think: https://docs.astral.sh/ruff/rules/#pyupgrade-up